debian/base/Dockerfile.rpm

```diff
@@ -0,0 +1,94 @@
+#
+# Copyright 2016 Confluent Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+FROM centos:centos7
+
+ARG COMMIT_ID=unknown
+LABEL io.confluent.docker.git.id=$COMMIT_ID
+ARG BUILD_NUMBER=-1
+LABEL io.confluent.docker.build.number=$BUILD_NUMBER
+
+ARG CONFLUENT_RPM_REPO="http://packages.confluent.io"
+# TODO: see about enabling this for rpms
+# ARG APT_ALLOW_UNAUTHENTICATED=false
+# #Set an env var so that it's available in derived images
+# ENV APT_ALLOW_UNAUTHENTICATED=${APT_ALLOW_UNAUTHENTICATED}
+
+MAINTAINER partner-support@confluent.io
+LABEL io.confluent.docker=true
+
+# # Python
+# # TODO: get this version
+# # ENV PYTHON_VERSION="2.7.9-1"
+ENV PYTHON_PIP_VERSION="9.0.1"
+
+# Confluent
+ENV SCALA_VERSION="2.11"
+ENV CONFLUENT_MAJOR_VERSION="3"
+ENV CONFLUENT_MINOR_VERSION="2"
+
+ENV CONFLUENT_PATCH_VERSION="1"
+ENV CONFLUENT_VERSION="$CONFLUENT_MAJOR_VERSION.$CONFLUENT_MINOR_VERSION.$CONFLUENT_PATCH_VERSION"
+
+ENV KAFKA_VERSION="0.10.2.1"
+
+# Zulu
+ENV ZULU_OPENJDK_VERSION="8-8.17.0.3"
+
+# This affects how strings in Java class files are interpreted.  We want UTF-8 and this is the only locale in the
+# base image that supports it
+ENV LANG="C.UTF-8"
+
+RUN echo "===> clean yum caches ....." \
+    && yum clean all
+
+RUN echo "===> Installing curl wget netcat python...." \
+    && yum install -y \
+                curl \
+                wget \
+                nc \
+                python
```

Comment:
wilco, but again, this is just c/p from the debs. maybe a second pr where i can go through and clean all this up? i'm not sure that we should be doing cleanup in feature addition prs